app/src/main/java/com/tari/android/wallet/ui/fragment/settings/allSettings/AllSettingsViewModel.kt

```diff
@@ -54,10 +59,15 @@ internal class AllSettingsViewModel : CommonViewModel() {
     private val _lastBackupDate: MutableLiveData<String> = MutableLiveData()
     val lastBackupDate: LiveData<String> = _lastBackupDate
 
+    private val _versionInfo: MutableLiveData<String> = MutableLiveData()
+    var versionInfo: LiveData<String> = _versionInfo
+
     init {
         component.inject(this)
         EventBus.backupState.subscribe(this) { backupState -> onBackupStateChanged(backupState) }
         checkStorageStatus()
+
+        _versionInfo.postValue("${networkRepository.currentNetwork!!.network.displayName} ${BuildConfig.VERSION_NAME} b${BuildConfig.VERSION_CODE}")
```

Comment:
We are using this string in few different places. Maybe it will be a good idea to create a dedicated formatter for this value to avoid code duplication.

Reply:
I extracted formatting and used it, but I would like drop unit test on that because in that case, I need to mock BuildConfig stuff but it totally not worth it